lib/galaxy/config/sample/object_store_conf.xml.sample

```diff
@@ -135,7 +135,7 @@
 <object_store type="aws_s3">
      <auth access_key="...." secret_key="....." />
      <bucket name="unique_bucket_name_all_lowercase" use_reduced_redundancy="False" />
-     <cache path="database/object_store_cache" size="1000" />
+     <cache path="database/object_store_cache" size="1000" cache_updated_data="True" />
```

Comment:
The `cache_updated_data` option should be documented, I think. The explanation you provided in the PR description is very good, but it will be helpful if it's more accessible (i.e., not archived in a PR description). Since we don't have a section on object store configuration in Galaxy's [documentation](https://docs.galaxyproject.org/) yet (or a schema file with all properties listed and explained), this sample config file is the best place for it. If it applies to any type of object store (I assume it does?), maybe add a brief description somewhere at the top of the file?